docs/rule/android_binary.soy

```diff
@@ -180,6 +180,21 @@ An <code>android_binary()</code> rule is used to generate an Android APK.
 {call android_common.deps_apk_arg /}
 
 {call buck.arg}
+  {param name : 'cpu_filters' /}
+  {param default : '[]' /}
+  {param desc}
+  The CPU architecture filter applied to the final apk. Could be a subset of <code>ARM</code>,
+  <code> ARMV7</code>, <code>X86</code>, <code>X86_64</code>, <code>MIPS</code>.
```

Comment:
I didn't spot this before because the line was so long :)

Instead of adding the space inside the `<code />` tag, you should put `{sp}` before it (or at the line before it if there is space).